sirius-web-tutorial-eclipsecon2023.adoc

```diff
@@ -511,30 +625,30 @@ image::images/studio_maker_domain_person_create_relation_children_set_properties
 Then, in the _View_ model, from the _Family Diagram Description_ object, create an _Edge Description_ and set its properties:
 
 * _Name_ to `Children Edge`
-* _Domain Type_ to empty value
+* _Domain Type_ to `tutorial::Person`
```

Comment:
`empty value` is the appropriate value since the `Is Domain Based Edge` is set to `false`, or am I missing something here?

Reply:
For relation-based edges, they are still associated to a semantic element so domain type is relevant. And with an empty value it works too, but we get a validation error which can be confusing IMO.